Controllers/TaskController.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace Controllers;
+
+use Models\TaskModel;
+
+class TaskController implements IController
+{
+    private $model;
+    private $fc;
+
+    public function __construct()
+    {
+        $this->model = new TaskModel();
+        $this->fc = FrontController::getInstance();
+    }
+
+    public function createTaskAction()
+    {
+        $imgName = "";
+        if ($_FILES['imageTask']["error"] == UPLOAD_ERR_OK) {
+            $imgName = rand() . $_FILES['imageTask']['name'];
+            move_uploaded_file($_FILES['imageTask']['tmp_name'], 'Media/images/tasks/' . $imgName);
+        }
+        $timeCreated = time();
+        $createdBy = $_POST["created_by"];
+        $executor = $_POST["executor"];
+        $task = $_POST["task"];
+        $status = "новое";
+        $timeStart = time() + (int)$_POST["time_start"] * 3600;
+        $timeEnd = $timeStart + (int)$_POST["time_end"] * 3600;
+        $approvedBy = "Не подтверждена";
+        $res = $this->model->createTask($timeCreated, $createdBy, $executor, $task, $status, $timeStart, $timeEnd, $approvedBy, $imgName);
```

Comment:
please call variable more clearly
$res? what is this? respawn? restart? resukablyat? or $result?